packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,328 @@ sidebar_position: 3
-- `reduceMotion` determines how the animation responds to the device's reduced motion accessibility setting.
+```js
+import { JumpingTransition } from 'react-native-reanimated';
 
-#### Example
+function App() {
+  return <Animated.View layout={JumpingTransition} />;
+}
+```
 
-<video
-  src="https://user-images.githubusercontent.com/48885911/134477879-e697aa03-b94d-4a65-b076-5ec3800310e3.mov"
-  controls="controls"
-  muted="muted"
-  height="600"
+### Modifiers
+
+```javascript
+JumpingTransition.duration(1000)
+  .delay(500)
+  .reduceMotion(ReduceMotion.Never)
+  .withCallback((finished) => {
+    console.log(`finished without interruptions: ${finished}`);
+  });
+```
+
+- `.duration(duration: number)` sets length of the animation (in milliseconds). Defaults to '300'.
+- `.delay(duration: number)` is the delay before the animation starts (in milliseconds). Defaults to `0`.
+- `.reduceMotion(reduceMotion: ReduceMotion)` determines how the animation responds to the device's reduced motion [accessibility](/docs/guides/accessibility) setting.
+- `.withCallback(callback: (finished: boolean) => void)` is the callback that will fire after the animation ends. Sets `finished` to `true` when animation ends without interruptions, and `false` otherwise.
+
+### Example
+
+<ExampleVideo
+  sources={{
+    android:
+      '/react-native-reanimated/recordings/layout_transitions/android_jumping_transition.mov',
+    ios: '/react-native-reanimated/recordings/layout_transitions/ios_jumping_transition.mov',
+  }}
 />
 
-### Jumping Transition
+## Curved Transition
 
-Jumping transition, component "jumps" to the new position.
+Curved transition enables to animate each position and dimension with different easing. It makes component movement path look curved.
 
-#### Modifiers
+### Reference
 
-- `duration` (in ms) default: 300
-- `delay` (in ms) default: 0
-- `withCallback` callback that will fire after the exit animation ends
-- `randomDelay` randomize delay of the animation between 0 and provided delay ( or 1000ms if delay not provided )
-- `reduceMotion` determines how the animation responds to the device's reduced motion accessibility setting.
+```js
+import { CurvedTransiton } from 'react-native-reanimated';
 
-#### Example
+function App() {
+  return <Animated.View layout={CurvedTransiton} />;
+}
+```
+
+### Modifiers
+
+```javascript
+CurvedTransiton.duration(1000)
+  .delay(500)
+  .easingX(Easing.in(Easing.exp))
+  .easingY(Easing.out(Easing.quad))
+  .easingWidth(Easing.in(Easing.ease))
+  .easingHeight(Easing.out(Easing.exp))
+  .reduceMotion(ReduceMotion.Never)
+  .withCallback((finished) => {
+    console.log(`finished without interruptions: ${finished}`);
+  });
+```
 
-<video
-  src="https://user-images.githubusercontent.com/48885911/134478585-71c601ae-137f-44e4-a58f-4bb6b60f6bdf.mov"
-  controls="controls"
-  muted="muted"
-  height="600"
+- `.duration(duration: number)` sets length of the animation (in milliseconds). Defaults to '300'.
```

Comment:
I'd add a note that length is shared for all the easings (isn't it?). Might go into remarks as well

Reply:
I'll put that into remarks, just as in EntryExit transition